packages/components/src/ButtonGroup/docs/ButtonGroup.stories.tsx

```diff
@@ -2,7 +2,7 @@ import React from "react"
 import { ComponentMeta, ComponentStory, Story } from "@storybook/react"
 import { Tooltip } from "@kaizen/draft-tooltip"
 import { StickerSheet } from "../../../../../storybook/components/StickerSheet"
-import { FilterButtonBase } from "../../FilterButton/components/FilterButtonBase"
+import { FilterButtonBase } from "../../FilterButton/_sub-components/FilterButtonBase"
```

Comment:
This probably makes more sense considering it's (at the moment) considered quite separate

```suggestion
import { FilterButtonBase } from "~components/FilterButton/_sub-components/FilterButtonBase"
```